test/raw-strings.test.ts

```diff
@@ -12,12 +49,8 @@ describe("decode with rawBinaryStringValues specified", () => {
   });
 
   it("decodes invalid UTF-8 string values as binary", () => {
-    const invalidUtf8String = Uint8Array.from([
-      61, 180, 118, 220, 39, 166, 43, 68, 219, 116, 105, 84, 121, 46, 122, 136, 233, 221, 15, 174, 247, 19, 50, 176,
-      184, 221, 66, 188, 171, 36, 135, 121,
-    ]);
     const encoded = Uint8Array.from([
-      196, 32, 61, 180, 118, 220, 39, 166, 43, 68, 219, 116, 105, 84, 121, 46, 122, 136, 233, 221, 15, 174, 247, 19, 50,
+      217, 32, 61, 180, 118, 220, 39, 166, 43, 68, 219, 116, 105, 84, 121, 46, 122, 136, 233, 221, 15, 174, 247, 19, 50,
```

Comment:
Unfortunately the old test case wasn't an encoded string, it was a binary string 😬